hasura/migrations/default/1676583921722_create_table_public_action/up.sql

```diff
@@ -9,6 +9,7 @@ CREATE TABLE "public"."action" (
   "app_id" varchar(50) NOT NULL,
   "max_accounts_per_user" integer NOT NULL DEFAULT 1,
   "max_verifications" integer NOT NULL DEFAULT 1,
+  "creation_mode" varchar(50) NOT NULL DEFAULT 'developer_portal',
```

Comment:
@paolodamico  Maybe we should add some comment for this field with the union of values it should contain?